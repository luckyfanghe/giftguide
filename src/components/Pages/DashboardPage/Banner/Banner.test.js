import React from "react";
import { render, fireEvent } from "@testing-library/react";
import renderer from "react-test-renderer";
import Banner from "./Banner";

it("Banner renders corrently", () => {
  const { queryByTestId } = render(<Banner />);

  expect(queryByTestId("Banner")).toBeTruthy();
});

describe("Banner", () => {
  test("Banner snapshot renders ", () => {
    const component = renderer.create(<Banner />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
