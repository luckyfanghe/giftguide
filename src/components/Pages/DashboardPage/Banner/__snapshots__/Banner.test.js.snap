// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Banner Banner snapshot renders  1`] = `
<div
  className="sc-AykKC cBnjZW"
  data-testid="Banner"
>
  <img
    alt="banner"
    className="sc-AykKD ceqHQh"
  />
  <button
    className="sc-AykKE kAoYeh"
    type="button"
  />
</div>
`;
